services/mobu/values-usdfprod.yaml

```diff
@@ -0,0 +1 @@
+cachemachineImagePolicy: "desired"
```

Comment:
You're enabling mobu for both environments but not running any checks, so it's sort of pointless. You probably want to enable at least the basic sanity check that idfdev runs.